README.md

```diff
@@ -56,11 +56,11 @@ Rob Ankeny ([ankenyr@gmail.com](mailto:ankenyr@gmail.com))
 Jason Benterou ([jason.benterou@gmail.com](mailto:jason.benterou@gmail.com))
 
 ## Resources
+
 - [Brief Overview (4 slides):](https://docs.google.com/present/embed?id=dhtc9k26_13cz9fphfb&autoStart=true&loop=true&size=1)
 - [Nanog49; Enterprise QoS](http://www.nanog.org/meetings/nanog49/presentations/Tuesday/Chung-EnterpriseQoS-final.pdf)
 - [Blog Post: Safe ACL Change through Model-based Analysis](https://tech.ebayinc.com/engineering/safe-acl-change-through-model-based-analysis/)
 - [Aerleon Slack](https://join.slack.com/t/aerleon/shared_invite/zt-1ngckm6oj-cK7yj63A~JgqjixEui2Vhw)
-- [#aerleon at NetworkToCode Slack](https://networktocode.slack.com/)
```

Comment:
Oops, we did intent to create such a channel. @ankenyr is creating one now. We would like to continue to direct people to NTC through the README.